frontend/packages/kubevirt-plugin/package.json

```diff
@@ -17,7 +17,38 @@
     "entry": "src/plugin.tsx",
     "integrationTestSuites": {
       "kubevirt": [
-        "integration-tests/tests/*.scenario.ts"
+        "integration-tests/tests/vm.setup.scenario.ts",
```

Comment:
similarly, I don't think this is very useful, since the tests are executed in alphabetical order, renaming the scenario files will by itself group them up.

Additionally, this will introduce the necessity to add a new entry when a new file is added. But only sometimes, because for example `vm.tab.somethingNew.scenario.ts` will be added automatically.
I think this is very error-prone and we shouldn't do it

Reply:
> I think renaming the tests is a good idea and it would help to navigate in the code easier.
> 
> I don't really agree with the idea of having the `vm.setup.scenario` for the following reasons:
> 
>     1. the time penalty for creating and deleting a container VM is very small.
> 
>     2. Some tests require having the VM in specific state (Running/stopped), we would still need to ensure that the VM is in the correct state, and stopping a VM takes more time that just deleting it, so this may take up more time in the end

Just see a group of tests can use a shared test VM, and it can make these tests look a bit tidy, so try to unique their test VM. And I was trying to add more tests to utilize the setup VM, since you don't think it's a good idea, I'll give up. Thanks.

Reply:
If it doesn't have `vm.setup.scenario.ts`, it doesn't need to specify one by one.